src/clj/rems/db/test_data_users.clj

```diff
@@ -39,7 +38,8 @@
    "organization-owner1" {:eppn "organization-owner1" :mail "organization-owner1@example.com" :commonName "Organization Owner 1"}
    "organization-owner2" {:eppn "organization-owner2" :mail "organization-owner2@example.com" :commonName "Organization Owner 2" :organizations [{:organization/id "organization2"}]}
    "owner" {:eppn "owner" :mail "owner@example.com" :commonName "Owner"}
-   "reporter" {:eppn "reporter" :mail "reporter@example.com" :commonName "Reporter"}})
+   "reporter" {:eppn "reporter" :mail "reporter@example.com" :commonName "Reporter"}
+   "elixir-alice" {:eppn "alice" :elixirId "elixir-alice" :mail "alice@elixir-europe.org" :commonName "Alice Applicant" :organizations [{:organization/id "default"}] :nickname "In Wonderland" :researcher-status-by "so"}})
```

Comment:
So what I was saying is that we probably can't control that `"elixirId"` comes from IdP. Likely it will be named `"sub"`. That's why we need the rename, if we internally want to have it be something else.